e2e/global-setup.ts

```diff
@@ -123,6 +124,17 @@ async function globalSetup(config: FullConfig) {
         headers,
     });
 
+    // set automatic replies setting (ooo) to true for notifications tests
+    resp = await adminContext.put('api/v4/config/patch', {
```

Comment:
What is needed for exactly?

Reply:
For the `user is OOO: no ringing, no desktop notification` test, automatic replies need to be enabled.

Reply:
Okay, should it perhaps be a per test setting to enable rather than global?

Reply:
Yep, makes sense. Done.

Reply:
To note this would likely cause all sorts of racy failures in the future if there's another unrelated test relying on the setting being on/off. That's on me for even suggesting it but I suppose we'll cross that bridge when needed.

Reply:
Right, it's not a very isolated test anymore.